di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -376,6 +378,17 @@ public AuthorizeHandler(
                         }
                     }
 
+                    String notBeforeFlag =
+                            queryParamsMap.value(CredentialIssuerConfig.VC_NOT_BEFORE_FLAG);
+
+                    Instant now = Instant.now();
+                    Long nbf = now.getEpochSecond();
+                    if (notBeforeFlag != null
+                            && notBeforeFlag.equals(
+                                    CredentialIssuerConfig.EXPIRY_FLAG_CHK_BOX_VALUE)) {
+                        nbf = getNbf(queryParamsMap);
+                    }
```

Comment:
I'd probably have included this check in the factored out method as well, but a matter of taste perhaps.

`EXPIRY_FLAG_CHK_BOX_VALUE` is also a bit suspicious, but not a massive deal!